org.osgi.test.cases.typedevent/src/org/osgi/test/cases/typedevent/junit/TestEvent.java

```diff
@@ -0,0 +1,16 @@
+package org.osgi.test.cases.typedevent.junit;
```

Comment:
All java files need to start with the license notice comment.

Reply:
This is not a junit test class, so it should not be in the junit package which is generally only the junit tests. So something like `org.osgi.test.cases.typedevent.dto`.